src/tests/ftest/container/per_server_fault_domain.yaml

```
@@ -0,0 +1,59 @@
+hosts:
+  test_servers: 4
+
+timeout: 300
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31416
+      log_file: daos_server_0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+        1:
+          class: nvme
+          bdev_list: ["aaaa:aa:aa.a"]
```

We should make use of the new storage auto feature:
```suggestion
      storage: auto
```

---

Done